python/lsst/ts/cbp/csc.py

```diff
@@ -115,21 +117,29 @@ async def telemetry(self):
             try:
                 self.log.debug("Begin sending telemetry")
                 await self.component.update_status()
-                if not self.evt_target.has_data:
-                    await self.evt_target.set_write(
-                        azimuth=self.component.azimuth,
-                        elevation=self.component.elevation,
-                        focus=self.component.focus,
-                        mask=self.component.mask,
-                        mask_rotation=self.component.mask_rotation,
-                    )
+                # if not self.evt_target.has_data:
```

Comment:
I see you moved this event to the `handle_summary_state` method. Apparently it needs to be sent before the `inPosition` event. Isn't it much simpler to send the event just before line 105?